code/game/objects/items/devices/object-in-a-box.dm

```diff
@@ -0,0 +1,29 @@
+/obj/item/device/object_in_a_box
```

Comment:
It's defined right above this
+if(ispath(being_built.build_path, /obj/structure)) // if we're making a structure, make an object-in-a-box item that dispenses said structure on use, or everything runtimes